src/TimeKeeper.php

```diff
@@ -31,29 +34,50 @@ class TimeKeeper
     public function __construct(DateTime $gameEpoch, int $gameOffsetSeconds, int $gameDaysPerDay)
     {
         $gameEpochCopy = clone($gameEpoch);
-        $this->adjustedEpoch = $gameEpochCopy->add(
-            $this->interval(0, 0, 0, 0, $gameOffsetSeconds)
-        );
+
+        if ($gameOffsetSeconds < 0) {
```

Comment:
Should this be supported? What's the user's expectation when providing a negative offset? It seems that a negative offset has no effect which is odd. I would prefer it throw an exception if its not going to do anything substantive.